actors/test/actor-tests/src/main/java/com/ea/orbit/actors/test/ActorBaseTest.java

```diff
@@ -198,7 +206,8 @@ protected boolean isIdle(Stage stage)
         {
             // this is very ad hoc, but should work for our tests, until execution changes.
             // for starters access to this map should be synchronized.
-            Map running = (Map) getField(getField(getField(stage, "execution"), "executionSerializer"), "running");
+            Map running = (Map) getField(getField(getField(stage, Stage.class, "execution"), Execution.class,
```

Comment:
bit more obvious like this and if we ever use an anonymous inner class to create ExecutionSerializer reflection will continue to work.